certgen/certProcessor/src/main/java/org/incredible/certProcessor/store/CloudStorage.java

```diff
@@ -29,19 +26,7 @@ public static String uploadFile(String container, String path, File file, boolea
                 objectKey,
                 Option.apply(isDirectory),
                 Option.apply(1),
-                Option.apply(retryCount), Option.empty());
+                Option.apply(retryCount), Option.apply(1));
```

Comment:
(last param)Option.apply(1) -> attempt

Reply:
can we take this value from env, bydefault it will read from constant but env can override it , if provided

Reply:
done